docusaurus/docs/reference/prompts.md

```diff
@@ -49,3 +49,27 @@ Give your plugin a name which helps identify its purpose.
 ### Enter your organization name (usually your Grafana Cloud org)
 
 Enter the name of your organization. This must be your [Grafana Cloud](https://grafana.com/signup/) organization to allow you to [sign](../publish-a-plugin/sign-a-plugin.md) and optionally [publish](../publish-a-plugin/publish-or-update-a-plugin.md) the plugin to the [catalog](https://grafana.com/grafana/plugins).
+Add a [GitHub workflow](/create-a-plugin/develop-a-plugin/set-up-github-workflows#the-compatibility-check-is-compatibleyml) to regularly check that your plugin is compatible with the latest version of Grafana.
+
+## Bypassing Prompts
+
+All the above prompts can be bypassed using cli arguments. To scaffold a plugin with cli arguments pass them to the create-plugin command like so:
+
+```
+npx @grafana/create-plugin --plugin-type="app" --plugin-name="myPlugin" --org-name="myorg" --backend
+```
+
+Scaffolding plugins using cli arguments not only makes it quicker to get started but allows you to run the tool in a non-interactive environment such as CI or to scaffold plugins with other tooling too.
+
+:::info
+To pass in false for a boolean create-plugin uses the naming convention `--no-<promptName>` (e.g. `--no-backend`).
+:::
```

Comment:
I feel like this is more useful in contribution guidelines/internal doc rather than needing to be user facing (particularly when we only have one prompt which follows this). Not a blocker, but would consider removing just because its arguably noise

Reply:
Tis done! 👍